guides/docusaurus-starter-guide/configuring.mdx

```diff
@@ -0,0 +1,85 @@
+
+### Tell CloudCannon about the data
+
+To configure the CloudCannon interface, you will need to tell CloudCannon how our your site is structured.
+For this we use [CloudCannon reader](https://github.com/CloudCannon/reader), an open-source NPM library used to integrate editing.
+You don't need to install this package, as CloudCannon will run it automatically as part of your build processes.
+
+Reader looks for a CloudCannon config file in the root of your site.
+You can configure almost all of the editing interfaces in CloudCannon using this file.
+
+Check out the [reader documentation](https://github.com/CloudCannon/reader) and define your collections.
+Once you have pushed your changes to CloudCannon your site will rebuild.
+After building, any changes to the interface that you configured in your config file will take effect.
+You will be able to:
+
+- Add, remove, edit and rename any collection item
+- Edit any data file
+
+You will have numerous editing options available to you:
+
+- Source Editor: edit your code directly within CloudCannon in a familiar-looking editor.
+- Data Editor: Edit your data and front matter in a recursive form full of customizable inputs.
+  - Add, remove, clone, rearrange and edit arrays
+  - Open objects and edit content to any depth
+  - Change data with date, color, number, URL, and more inputs
+- Content Editor: Edit your markdown files and data at the same time. A big content block for your markdown and an embedded data editor on the side. This is perfect for a combination of markdown and front matter files.
+- Visual Editor: A combination editor with your live site on one side and the data editor on the other side. The rest of this tutorial will focus on integrating the two sides more.
+
+### Configuring CloudCannon
+
+With your collections configured in CloudCannon, you can focus on polishing the experience for your editors.
+A good place to start is expanding on your collection configuration. This allows you to do things like:
+
+1. Change the icons, names, descriptions, and documentation links in the collections interface.
+2. Define which editors can be used to edit each collection item.
+3. Manage the order of collections in the sidebar and how they are grouped (e.g. Blogging &gt; Posts).
+4. And, Show and hide collections entirely.
+
+The global configuration file is one of many places that configuration can exist. Some options can exist only in the global config. Others options can be configured in a number of places; these options always start with an `_` character.
+These options cascade like CSS: the more specific the options are, the more they take precedence. From least important to most important, the order is:
+
+1. Global configuration.
+2. Collection configuration.
+3. Front matter.
+4. Containing structure.
+
+### Configuring inputs
```

Comment:
A couple links to docs about _inputs and _structures could be helpful here!

Reply:
Done!